helm/mojaloop-barebone/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v2
+name: ph-ee-barebone
+description: PaymentHub EE Barebone Edition
+
+type: application
+version: 0.1.0
+appVersion: 1.16.0
+
+dependencies:
+- name: ph-ee-engine
+  repository: https://fynarfin.io/images/
+  version: 1.0.0-SNAPSHOT
```

Comment:
use latest  version.

Reply:
I have just renamed Mojaloop-barebone. Every thing is as it is.

Reply:
the Idea is to use this chart for development. The chart has to work with latest changes

Reply:
My task is to work on paymenthub barebone chart. Its a mojaloop-barebone chart. All the changes in the Mojaloop chart are as it is. Only I have renamed it from paymenthub to differentiate between paymenthub barebone and mojaloop barebone.